jigoshop-software.php

```diff
@@ -1414,16 +1453,119 @@ function ajax_jgs_lost_license() {
 			exit;
 		}
 
+
+		/**
+		 * Unsubscribe from activation notification emails.
+		 *
+		 * @since 2.7
+		 * @author Anton Iancu <anton.iancu@gmail.com>
+		 */
+
+		function ajax_jgs_activation_unsubscribe() {
+				//reset message and reponse vars
+				$message = null;
+				$response = null;
+				$unsubscribed = null;
+
+				//	 nonce verification
+				if ( $_POST['jgs_activation_unsubscribe_nonce'] && !wp_verify_nonce( $_POST['jgs_activation_unsubscribe_nonce'], 'jgs_activation_unsubscribe' ) ) $messages['nonce'] = __( 'An error has occurred, please try again', 'jigoshop-software' );
```

Comment:
this could render a notice without a ` ! empty( $_POST['jgs_activation_unsubscribe_nonce']`